cmd/ethrex/initializers.rs

```diff
@@ -64,9 +64,21 @@ pub fn init_metrics(opts: &Options, tracker: TaskTracker) {
     tracker.spawn(metrics_api);
 }
 
+/// Opens a New or Pre-exsisting Store and loads the initial state provided by the network
 pub async fn init_store(data_dir: &str, network: &str) -> Store {
```

Comment:
this changed: https://github.com/lambdaclass/ethrex/pull/2949/files#diff-7b682b3249e0f8b4addfca5976e806ad6ff115fe2d15def78fb2d500e571ec44R65

Reply:
Updated by merge commit: [283920e](https://github.com/lambdaclass/ethrex/pull/2995/commits/283920ea2af443536eae78d8c06392a1291b6635)